docs/i2c.rst

```diff
@@ -0,0 +1,43 @@
+I²C
+***
+
+.. py:module:: microbit.i2c
+
+The ``i2c`` module lets you communicate with devices connected to your board
+using the I²C bus protocol. There can be multiple slave devices connected at
+the same time, and each one has its own unique address, that is either fixed
+for the device or configured on it. Your board acts as the I²C master.
```

Comment:
I think that it is safe to assume that a user who attempts to use an I²C device already has some familiarity with the protocol, and if not, should familiarize with it before actually connecting anything. Renaming it to "controller" and "controlled" will only add confusion, because the official documentation for the device is going to use the established and standard terms "master" and "slave", and the user will have to guess that this is what we mean.